content/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent.md

```diff
@@ -88,14 +88,19 @@ Before adding a new SSH key to the ssh-agent to manage your keys, you should hav
         UseKeychain yes
         IdentityFile ~/.ssh/id_rsa
       ```
+  {% note %}
 
+  **Note:** If you are not using the passphrase and if you get an error then try removing the UseKeychain line.
+  
+  {% endnote %}
```

Comment:
To render correctly, the note box needs a blank line above the opening tag, and more indentation. See [the staging deployment](https://docs-908--patch-1.herokuapp.com/en/free-pro-team@latest/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#adding-your-ssh-key-to-the-ssh-agent). I'd also suggest simplifying the advice about `UseKeychain`:
 
```suggestion

     {% note %}

     **Note:** If you chose not to add a passphrase to your key, you should omit the `UseKeychain` line.
  
     {% endnote %}
```